java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/explorer/TreeItem.java

```diff
@@ -56,6 +57,11 @@
     public String resourceUri;
     // ?: string | MarkdownString | undefined
     public Object tooltip;
+    
+    public static class IconDescriptor {
+        public URI baseUri;
+        public String[] composition;
```

Comment:
Please add a comment describing the meaning of `composition` array.

Reply:
Fixed in b20f0d6